download_tokenizer_files.py

```diff
@@ -0,0 +1,22 @@
+# /// script
```

Comment:
I am guessing one time job hence we are not migrating, wondering if we should still for consistency?